nisystemlink/clients/notebook/models/_query_execution_request.py

```diff
@@ -0,0 +1,98 @@
+from enum import Enum
+from typing import List, Optional
+
+from nisystemlink.clients.core._uplink._json_model import JsonModel
+
+
+class ExecutionSortField(str, Enum):
+    """Possible fields used to sort executions."""
+
+    NOTEBOOK_ID = "NOTEBOOK_ID"
+
+    EXECUTION_HASH = "EXECUTION_HASH"
+
+    TIMEOUT = "TIMEOUT"
+
+    QUEUED_AT = "QUEUED_AT"
+
+    STARTED_AT = "STARTED_AT"
+
+    COMPLETED_AT = "COMPLETED_AT"
+
+    STATUS = "STATUS"
+
+
+class ExecutionField(str, Enum):
```

Comment:
You're adding `last_updated_by` and `retry_count` as fields, and presumably those can be included in a projection. If so you'll want to include those two fields here.